src/test/kotlin/no/nav/bidrag/bidragskalkulator/controller/BeregningControllerTest.kt

```diff
@@ -61,13 +62,50 @@ class BeregningControllerTest: AbstractControllerTest() {
             .andExpect(jsonPath("$.errors[0]").value("barn: Liste over barn kan ikke være tom"))
     }
 
+    @Test
+    fun `skal returnere 400 hvis dittBoforhold mangler for PLIKTIG`() {
+        val ugyldigRequest = mockGyldigRequest.copy(dittBoforhold = null)
+
+        postRequest("/api/v1/beregning/barnebidrag", ugyldigRequest, gyldigOAuth2Token)
+            .andExpect(status().isBadRequest)
+            .andExpect(jsonPath("$.errors[0]").value("'dittBoforhold' må være satt fordi forespørselen inneholder minst ett barn der du er bidragspliktig."))
+    }
+
+    @Test
+    fun `skal returnere 400 hvis medforelderBoforhold mangler for MOTTAKER`() {
+        val ugyldigRequest = mockGyldigRequest.copy(
+            barn = listOf(BarnDto(ident = Personident(personIdent), samværsklasse = Samværsklasse.SAMVÆRSKLASSE_0, bidragstype = BidragsType.MOTTAKER))
+        )
+
+        postRequest("/api/v1/beregning/barnebidrag", ugyldigRequest, gyldigOAuth2Token)
+            .andExpect(status().isBadRequest)
+            .andExpect(jsonPath("$.errors[0]").value("'medforelderBoforhold' må være satt fordi forespørselen inneholder minst ett barn der du er bidragsmottaker."))
+    }
+
+    @Test
+    fun `skal returnere 400 hvis begge boforhold mangler`() {
```

Comment:
Kan være fint med litt mer beskrivelse her 
```suggestion
    fun `skal returnere 400 hvis bruker er både bidragsmottaker og bidragspliktig, og begge boforhold mangler`() {
```

Reply:
Skulle vi hatt en test for at bruker er både mottaker og pliktig, men kun én av de er satt?